From b84878a29d681598370e1a3e79e541548cb7db6f Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Mon, 7 Mar 2022 12:06:14 +0100
Subject: [PATCH 088/106] Revert "bfq: remove unnecessary
 BFQ_DEFAULT_GRP_IOPRIO"

This reverts commit e4cb4938fd7091cc8f7f720f309de68777dcaea0.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 block/bfq-iosched.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/bfq-iosched.h b/block/bfq-iosched.h
index b952ad63c..bbd9f64b3 100644
--- a/block/bfq-iosched.h
+++ b/block/bfq-iosched.h
@@ -22,6 +22,7 @@
 #define BFQ_DEFAULT_QUEUE_IOPRIO	4
 
 #define BFQ_WEIGHT_LEGACY_DFL	100
+#define BFQ_DEFAULT_GRP_IOPRIO	0
 #define BFQ_DEFAULT_GRP_CLASS	IOPRIO_CLASS_BE
 
 #define MAX_BFQQ_NAME_LENGTH 16
-- 
2.35.1.354.g715d08a9e5

